docs/AboutUs.md

```diff
@@ -9,15 +9,15 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 
 ## Project team
 
-### John Doe
+### Delton Cheng
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/deltoncheng.png" width="200px">
 
-[[homepage](http://www.comp.nus.edu.sg/~damithch)]
-[[github](https://github.com/johndoe)]
-[[portfolio](team/johndoe.md)]
+[[github](https://github.com/DeltonCheng)]
+[[portfolio]()]
```

Comment:
Should you link the portfolio to your PPP file?

Reply:
Linked the PPP in #31, will deconflict when merging. Thanks for the reminder :)

Reply:
Ignore previous, have included the portfolio in this pull request as well. Thanks :)